resources/dicts/thoughts/alive/medicine_cat.json

```diff
@@ -86,7 +143,8 @@
     {
         "id": "charismatic_medicine_cat",
         "thoughts": [
-            "Is doing {PRONOUN/m_c/poss} daily checkup on the elders"
+            "Is doing {PRONOUN/m_c/poss} daily checkup on the elders",
+	    "Easily persuades another Clan's medicine cat to gift c_n herbs"
```

Comment:
We have moon events that mirror this and are also capable of actually changing the clan's herb supply... perhaps we could reword this a bit? Perhaps into "Is considering the best way to ask another Clan for herbs"?  or "Is coming up with just the right words to ask another Clan's medicine cat for extra herbs"